scripts/ai/mcp/support/uv_resolver.ps1

```
@@ -264,27 +264,74 @@ raise SystemExit(subprocess.call(sys.argv[1:]))
     }
 }
 
+function Add-BioetlFlattenedStrings {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Add-BioetlFlattenedStrings' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Add-BioetlFlattenedStrings'

(PSUseSingularNouns)